boards/shields/g1120b0mipi/g1120b0mipi.overlay

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	aliases {
+		kscan0 = &kscan_input;
+	};
+
+	chosen {
+		zephyr,display = &rm67162_g1120b0mipi;
+		zephyr,keyboard-scan = &kscan_input;
+	};
+
+	en_mipi_display_g1120b0mipi: enable-mipi-display {
+		compatible = "regulator-fixed";
+		regulator-name = "en_mipi_display";
+		enable-gpios = <&nxp_mipi_connector 32 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+	};
+};
+
+&nxp_mipi_i2c {
+	status = "okay";
+	ft3267_g1120b0mipi: ft3267@38 {
```

Comment:
I would drop this nodelabel if you think it won't be needed in the board file (which it should not since you are referring to the pins through the nexus node)